CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to the IBM® Db2® Plug-in for Zowe CLI will be documented i
 ## Recent Changes
 
 - BugFix: Updated minimum supported version of Node from 18 to 20. Added Node 24 support. [#190](https://github.com/zowe/zowe-cli-db2-plugin/pull/190)
+  - Note: Introduces unavoidable breaking changes to the DB2 plug-in APIs as a result of breaking type changes in the underlying `ibm_db` component.
+    - Deprecated the `DB2Constants` class. Use `ibm_db`'s `FETCH_OBJECT` and `FETCH_ARRAY` instead.
```

Comment:
feels weird to use a possessive here. can we do something like:

Deprecated the `DB2Constants` class. Use `FETCH_OBJECT` and `FETCH_ARRAY` from the `ibm_db` class/group/property/WHAT instead.